raspbian/Dockerfile

```diff
@@ -38,16 +39,30 @@ RUN \
     && mkdir -p /etc/fix-attrs.d \
     && mkdir -p /etc/services.d \
     \
+    && git clone --depth 1 -b v${MIMALLOC_VERSION} \
+        "https://github.com/microsoft/mimalloc" /usr/src/mimalloc \
+    && mkdir -p /usr/src/mimalloc/build \
+    && cd /usr/src/mimalloc/build \
+    && cmake .. \
+    && make -j "$(nproc)" \
+    && make install \
+    \
     && curl -L -f -s -o /usr/bin/tempio \
-        "https://github.com/home-assistant/tempio/releases/download/${TEMPIO_VERSION}/tempio_armhf" \
+        "https://github.com/home-assistant/tempio/releases/download/${TEMPIO_VERSION}/tempio_${BUILD_ARCH}" \
```

Comment:
`BUILD_ARCH` isn't requested for yet?

Need to be added to the args lists above

Reply:
right can be static like for s6-overlay

Reply:
Actually, it cannot?

There is a single build file, for multiple architectures in this case. I think this is a bug?
This file is used by amd64 as well.

Reply:
No, we support only amrhf. But I think we should drop that image anyway since debian support armhf

Reply:
Oh! Right 🤦